src/lib/pages/frontsite/ErrorsPage/ErrorsPage.js

```diff
@@ -17,6 +17,13 @@ export class ErrorsPage extends Component {
         return <InternalServerError />;
       } else if (params.errorCode === 404) {
         return <NotFound />;
+      } else if (params.errorCode === 410) {
+        return (
+          <NotFound
+            title="Resource No Longer Available"
+            message="The requested content has been removed by the library manager"
```

Comment:
suggestion: Just a personal preference.
```suggestion
            message="The requested content has been removed."
```